.vscode/extensions.json

```diff
@@ -4,7 +4,8 @@
     "streetsidesoftware.code-spell-checker",
     "esbenp.prettier-vscode",
     "dbaeumer.vscode-eslint",
-    "astro-build.astro-vscode"
+    "astro-build.astro-vscode",
+    "ms-playwright.playwright"
```

Comment:
The extension is pretty neat

https://github.com/iTwin/iTwinUI/assets/9084735/7be6e5e4-5c81-40a7-a188-37c86b89316a

Reply:
Wow, that seems incredibly useful for debugging tests.